src/m365/spo/commands/tenant/tenant-applicationcustomizer-get.ts

```diff
@@ -0,0 +1,146 @@
+import { Logger } from '../../../../cli/Logger';
+import request, { CliRequestOptions } from '../../../../request';
+import { formatting } from '../../../../utils/formatting';
+import { spo } from '../../../../utils/spo';
+import GlobalOptions from '../../../../GlobalOptions';
+import { urlUtil } from '../../../../utils/urlUtil';
+import { validation } from '../../../../utils/validation';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { ListItemInstanceCollection } from '../listitem/ListItemInstanceCollection';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  title?: string;
+  id?: string;
+  clientSideComponentId?: string;
+}
+
+class SpoTenantApplicationCustomizerGetCommand extends SpoCommand {
+  public get name(): string {
+    return commands.TENANT_APPLICATIONCUSTOMIZER_GET;
+  }
+
+  public get description(): string {
+    return 'Get an application customizer that is installed tenant wide';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        title: typeof args.options.title !== 'undefined',
+        id: typeof args.options.id !== 'undefined',
+        clientSideComponentId: typeof args.options.clientSideComponentId !== 'undefined'
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-t, --title [title]'
+      },
+      {
+        option: '-i, --id [id]'
+      },
+      {
+        option: '-c, --clientSideComponentId  [clientSideComponentId]'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        if (args.options.id && !validation.isValidGuid(args.options.id)) {
+          return `${args.options.id} is not a valid GUID`;
+        }
+
+        if (args.options.clientSideComponentId && !validation.isValidGuid(args.options.clientSideComponentId)) {
+          return `${args.options.clientSideComponentId} is not a valid GUID`;
+        }
+
+        return true;
+      }
+    );
+  }
+
+  #initOptionSets(): void {
+    this.optionSets.push({ options: ['title', 'id', 'clientSideComponentId'] });
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      const spoUrl: string = await spo.getSpoUrl(logger, this.debug);
+      console.log(`${spoUrl}/_api/SP_TenantSettings_Current`);
+      const requestOptions: any = {
+        url: `${spoUrl}/_api/SP_TenantSettings_Current`,
+        headers: {
+          accept: 'application/json;odata=nometadata'
+        }
+      };
+
+      const res: string = await request.get(requestOptions);
+      const json = JSON.parse(res);
```

Comment:
Let's not parse JSON manually here. Instead, let's set the response type in requestOptions to JSON and get back an actual object. Also, since we're referring to a property from `json`, let's define it as a proper type rather than using `any`